donate/thunderbird/templates/pages/core/landing_page.html

```diff
@@ -0,0 +1,8 @@
+
+{% extends "pages/core/landing_page_master.html" %}
+
+{% block notice_bar %}
+{% if request.GET.tbdownload == 'true' %}
```

Comment:
do we want `true` to be case-sensitive?

Reply:
Depends: is this a string or a boolean? Or, is the only thing we really care about whether that argument exists? E.g. `{% if 'tbdownload" in request.GET %} ... {% endif %}`

Reply:
Forgot to include the link to the original [ticket](https://github.com/MozillaFoundation/Thunderbird-Migration/issues/30) (I have also updated PR description to include it).

> Depends: is this a string or a boolean? 

I'll have to defer this quesiton to @alanmoo

Reply:
Just check if it exists.

Reply:
I thought I responded to this comment, but just check if the argument exists.

Reply:
ah, yeah in that case we can use `{% if "tbdownload" in request.GET %} ... {% endif %}`